Makefile

```diff
@@ -12,8 +12,8 @@ es: pex
 pt-BR: pex
 	PEX_MODULE=contentpacks ./makecontentpacks ka-lite pt-BR 0.16 --out=out/langpacks/pt-BR.zip --no-assessment-resources --videolang=pt
 
-sw: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite sw 0.16 --out=out/langpacks/sw.zip --no-assessment-resources --videolang=sw
+swa: pex
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite swa 0.16 --out=out/langpacks/swa.zip --no-assessment-resources --videolang=swa --contentlang=swa
```

Comment:
Ok @benjaoming will stick on your idea. 
As of now what I am trying to do is to build the `swa` language pack so that I can debug/compare it on the `sw` language pack.